src/components/suggestions.ts

```diff
@@ -39,6 +39,11 @@ export const initSuggestionsTable = async (db: Database): Promise<void> => {
   );
 };
 
+// Get id and suggestion from a list of Suggestions
+export const getSuggestionPrintout = async (suggestions: Suggestion[]): Promise<string> => {
+  return suggestions.map((suggestion) => '**' + suggestion['id'] + '** | ' + suggestion['suggestion'] + '\n').join('');
```

Comment:
if you just want a newline between each suggestion, you could just do `.join('\n')` and remove the newline in the map function